Makefile

```diff
@@ -89,7 +89,7 @@ endif
 	@echo "Linting with golint"
 	$(shell golint $(shell find . -maxdepth 1 -type d \( ! -iname ".git" ! -iname "vendor" \)) )
 
-build: deps build_image _run_ci _push_image
+build: deps build_image _run_ci 
```

Comment:
The `_push_image` target needs to take image name as @kmova mentioned.

Reply:
Added the arch image in push script.